hass_nabucasa/iot_base.py

```diff
@@ -215,6 +215,7 @@ async def _handle_connection(self) -> None:
                 try:
                     msg = await self.client.receive(55)
                 except asyncio.TimeoutError:
+                    self._logger.warning("Timeout while waiting to receive message")
```

Comment:
```suggestion
                    self._logger.info("Timeout while waiting to receive message")
```
I'm unsure if this should be info or a warning.
It is something unexpected, so generally, a warning makes sense.
However, warnings tend to end up in creating issues against this lib or the cloud integration.
And this is not (generally) directly linked to that but something else holding things up on the instance so info might be better to still have the context while troubleshooting.

Reply:
Sound good. I'd add a comment why we set it to info and not warning.